components/sidebar_right/index.ts

```diff
@@ -59,4 +63,4 @@ function mapDispatchToProps(dispatch) {
     };
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(SidebarRight);
+export default connect(mapStateToProps, mapDispatchToProps)(SidebarRight as unknown as React.ComponentClass);
```

Comment:
why not simply do an `as React.ComponentClass`?

Reply:
I did the same before this hack but it was still showing errors by type-check and I did some research and I get to this hack.

Reply:
```
Conversion of type 'typeof SidebarRight' to type 'ComponentClass<{}, any>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.

Conversion of type 'typeof SidebarRight' to type 'ComponentClass<{}, any>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Types of parameters 'props' and 'props' are incompatible.
    Type '{}' is missing the following properties from type 'Props': isExpanded, isOpen, currentUserId, searchVisible, and 2 more.
```

I am still getting this error for `unknown` type.